app/components/live-join.hbs

```diff
@@ -0,0 +1,20 @@
+<section data-test-card class='card live__join'>
+  <h3 data-test-card-title class='card__title'>Join the Event</h3>
```

Comment:
Can we remove the top bottom margin for this heading ?

Reply:
Yes sure. I was using an existing class so didn't mess with it.